elixir/lesson2/tutorial.md

```diff
@@ -0,0 +1,86 @@
+---
+layout: page
+title: Interactive Elixir
+---
+## Enter IEx
+
+**IEx** is Elixir's interactive shell and a powerful ally to your journey
+of learning Elixir. You will also hear the term *REPL* used to refer to
+IEx.
+
+REPL stands for (__R__)ead - (__E__)valuate - (__P__)rint - (__L__)oop. To
+understand what this means lets look at an example.
+
+Open your terminal (mac) or command prompt (windows) and type `iex`. You
+should see the following output (numbers may vary):
+
+```erlang
+Erlang/OTP 19 [erts-8.1] [source] [64-bit] [smp:8:8] [async-threads:10] [hipe] [kernel-poll:false] [dtrace]
+
+Interactive Elixir (1.3.2) - press Ctrl+C to exit (type h() ENTER for help)
+iex(1)>
+```
+
+Now type the expression `1 + 1` and press enter. You should see `2` printed
+on the screen. You may have noticed that you can type another expression
+now, for example `1 - 1`. A REPL therefore _reads_ an expression like `1 +
+1`, it _evaluates_ it (2), it _prints_ it to the screen and then it _loops_
+back to the beginning so you can type expressions again.
+
+## Modules
+
+In our first lesson we talked about the idea of _functions_ and _data_. In
+functional languages it is common to group related functions into something
+called a _module_. A module can be thought of as a container for actions
+that act upon similar pieces of information.
+
+For example in Elixir we have a `Integer` module which contains functions
+that act upon integers.
+
+Try the following inside IEx:
+
+```erlang
+Integers.digits(123)
```

Comment:
do you mean `Integer.digits(123)`? `Integers.digits()` gives me a `UndefinedFunctionError`